backend/src/main/kotlin/be/osoc/team1/backend/security/EmailUtil.kt

```diff
@@ -0,0 +1,67 @@
+package be.osoc.team1.backend.security
+
+import org.springframework.mail.SimpleMailMessage
+import org.springframework.mail.javamail.JavaMailSender
+import org.springframework.mail.javamail.JavaMailSenderImpl
+import java.util.Properties
+import java.util.UUID
+
+/**
+ * This object contains every function needed to make and send emails.
+ */
+object EmailUtil {
```

Comment:
I see you moved most of the stuff into the forgot password service, looks very good to me. This separation here is also nice but I have just a last little request.

This works just fine, but could probably just be a regular service class with the @Service annotation, and you can then just inject it in `ForgotPasswordService`. There's not really much reason to have this be a static object instead of a managed bean. This should only require a few lines to be changed.

So you would have basically:
```kotlin
@Service
class EmailService {
```

And then in the `ForgotPasswordService`.
```kotlin
 @Service
class ForgotPasswordService(private val userRepository: UserRepository, private val passwordEncoder: PasswordEncoder, private val emailService: EmailService) {
```